conformance/tests/httproute-redirect-host-and-status.go

```diff
@@ -50,7 +50,7 @@ var HTTPRouteRequestRedirect = suite.ConformanceTest{
 				StatusCode: 302,
 			},
 			RedirectRequest: &roundtripper.RedirectRequest{
-				Hostname: "example.org",
+				Host: "example.org",
```

Comment:
The `HTTPRequestRedirectFilter` still includes the related field `Hostname`, but this is now `Host` so we have two different names for the same concept in similar areas. Can you help me to better understand the decision to change this attribute?

Reply:
I still feel a bit odd about changing this name here, but I wont consider it a blocker.